package.json

```diff
@@ -129,6 +129,7 @@
     "node-fetch": "^2.6.7",
     "node-jose": "^2.2.0",
     "normalize-path": "^3.0.0",
+    "npm": "^9.0.0",
```

Comment:
@AyushDhimanIITR Is this added package related to the current fix?

Reply:
No, i'll correct it and then raise the PR

Reply:
Hey @morhag90 , I have done the changes, let me know if there's anything left